src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1508,5 +1508,9 @@ internal enum ErrorCode
         ERR_RefReturnReadonlyNotField2 = 8411,
         ERR_ExplicitReservedAttr = 8412,
         ERR_TypeReserved = 8413,
+
+        ERR_FieldsInRoStruct = 8514,
```

Comment:
> ERR_FieldsInRoStruct = 8514, [](start = 7, length = 29)

Please add a PROTOTYPE comment for us to clean up the gap here when we merge the feature.